build.sbt

```diff
@@ -108,11 +108,14 @@ lazy val macros = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     name    := "enumeratum-macros",
     version := Versions.Macros.head,
     crossScalaVersions := scalaVersionsAll, // eventually move this to aggregateProject once more 2.13 libs are out
-    libraryDependencies += {
+    libraryDependencies ++= {
       if (scalaBinaryVersion.value == "3") {
-        "org.scala-lang" %% "scala3-compiler" % scalaVersion.value % Provided
+        Seq(
+          "org.scala-lang" %% "scala3-compiler" % scalaVersion.value % Provided,
+          "org.scala-lang" % "scala-reflect" % scala_2_13Version,
```

Comment:
Curious if there's a way to make this additional dependency
* Contingent on actually using the mix mode macro
* Only a compile time constraint?

Reply:
Unfortunately I think only the consumer of the library knows if the mixed mode macro is being used, from here, sbt only knows that it's a scala3 build.

I believe we could mark this as `Provided` too, but then all consumers would need to explicitly provide it. Marking it provided here means I also need to add it as Provided in `core`. It would only need to be added as a normal/runtime dep if the project was using the mixed mode because I think this is only needed for the scala 2 portion of the build.

Reply:
@lloydmeta let me know if you have a preference here.